Packs/CommonScripts/Scripts/SetGridField/SetGridField.py

```diff
@@ -21,12 +21,30 @@ def normalized_string(phrase: str) -> str:
     Examples:
         >>> normalized_string("TestWord")
         "testword"
-        >>> normalized_string("hello_world")
-        "hello_world"
+        >>> normalized_string("helloworld")
+        "helloworld"
     """
     return phrases_case.camel(phrase).replace("'", "").lower()
 
 
+def normalized_column_name(phrase: str) -> str:
+    """ Normalize columns or Grid to connected word in lower-case.
+
+    Args:
+        phrase: Phrase to normalize.
+
+    Returns:
+        str: Normalized phrase.
+
+    Examples:
+        >>> normalized_string("Test Word!@#$%^&*()-=+")
+        "testword"
+        >>> normalized_string("hello_world@")
+        "hello_world"
+    """
+    return re.sub(r'[^a-zA-Z\d_]', '', phrase[:255]).lower()
```

Comment:
Please add a note that this matches the logic of `stripToClumnName()` from the client's `strings.js`.

Reply:
should the slicing happen before the replacement?
currently, the final result may be less than 255 letters.

Reply:
yes, see [here](https://code.pan.run/xsoar/web-client/-/blob/master/src/util/strings.js#L461-466)